dev-manuals/test-plans/minigames/crosswordpuzzle-frontend/plans/u-crosswordpuzzle-3-finish-game.md

```diff
@@ -0,0 +1,38 @@
+# Crosswordpuzzle finish game (`u.crosswordpuzzle-3`)
+
+Version: v1.0, 2022-08-12, created test plan \
+Author: Aaron Schmid
+
+
+## Description
+
+This use case verifies that the game can be finished.
+
+## Precondition
+
+A configuration has been stored in the backend and selected in the frontend.
+
+## Postcondition
+
+A modal is shown where you result is displayed.
+
+## Typical procedure
+
+1. Open the crossword-puzzle
+2. Press the `start` button
+3. Input the correct answers in the crosswordpuzzle
+4. Press the `Evaluate` button
+5. A modal is shown that says `Congratulations!` `Everything right!`
```

Comment:
A modal? Not a toast?
```suggestion
5. A success message is shown
```

Reply:
After evaluationg a modal with the restart and the close button shows up, where the result is displayed.